### PR TITLE
Integrate Akka-base processors server

### PR DESCRIPTION
**Pull request by hickst**

### CODE REVIEW

File: corenlp/src/main/scala/org/clulab/processors/coserver/ProcessorCoreServer.scala [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+import com.typesafe.scalalogging.LazyLogging
+
+import akka.actor._
+import akka.actor.SupervisorStrategy._
+import akka.routing._
+import akka.event.Logging
+
+import org.clulab.processors._
+import org.clulab.processors.bionlp._
+// import org.clulab.processors.clulab._       // TBD: enable when merged
```
Comment (myedibleenso) [inline corenlp/src/main/scala/org/clulab/processors/coserver/ProcessorCoreServer.scala]: >TBD: enable when merged

What is this about?

File: corenlp/src/main/scala/org/clulab/processors/coserver/ProcessorActor.scala [hunk 2]
```diff
@@ -0,0 +1,186 @@
+package org.clulab.processors.coserver
```
Comment (myedibleenso) [inline corenlp/src/main/scala/org/clulab/processors/coserver/ProcessorActor.scala]: Forgive me for being dense, but what is `coserver` supposed to suggest (specifically the "co")?

File: corenlp/src/main/scala/org/clulab/processors/coserver/ProcessorCoreServer.scala [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+import org.clulab.processors.corenlp._
+import org.clulab.processors.fastnlp._
+import org.clulab.processors.shallownlp._
+
+/**
+  * Application to wrap and serve various Processors capabilities.
+  *   Written by: Tom Hicks. 6/5/2017.
+  *   Last Modified: Update for removal of Malt.
+  */
+object ProcessorCoreServer extends App with LazyLogging {
```
Comment (myedibleenso) [inline corenlp/src/main/scala/org/clulab/processors/coserver/ProcessorCoreServer.scala]: Why is this a runnable?

File: corenlp/src/test/scala/org/clulab/processors/coserver/TestProcessorCoreServerConfigs.scala [hunk 4]
```diff
@@ -0,0 +1,95 @@
+package org.clulab.processors.coserver
+
+import org.scalatest.{ Matchers, FlatSpec }
+
+import com.typesafe.config.{ Config, ConfigValueFactory, ConfigFactory }
+import com.typesafe.scalalogging.LazyLogging
+
+import org.clulab.processors._
+import org.clulab.processors.bionlp._
+// import org.clulab.processors.clulab._       // TBD: enable when merged
```
Comment (myedibleenso) [inline corenlp/src/test/scala/org/clulab/processors/coserver/TestProcessorCoreServerConfigs.scala]: ?

File: corenlp/src/main/scala/org/clulab/processors/coserver/ProcessorCoreServer.scala (hunk 1 above)
Comment (hickst) [inline corenlp/src/main/scala/org/clulab/processors/coserver/ProcessorCoreServer.scala]: These were markers for the next task but I realized later it didn't need doing. Thanks for spotting them.

File: corenlp/src/main/scala/org/clulab/processors/coserver/ProcessorActor.scala (hunk 2 above)
Comment (hickst) [inline corenlp/src/main/scala/org/clulab/processors/coserver/ProcessorActor.scala]: processors core server -- I couldn't think of a pithy name for this concept.

File: corenlp/src/main/scala/org/clulab/processors/coserver/ProcessorCoreServer.scala (hunk 3 above)
Comment (hickst) [inline corenlp/src/main/scala/org/clulab/processors/coserver/ProcessorCoreServer.scala]: Removed.

**Pull request merged**